### DISCUSSION TITLE
Error restoring backup - version 3.1.3.1

### DISCUSSION THREAD

**Discussion by vura** [nilsteampassnet/TeamPass#4511] [Q&A]
Hii,
I had an error when I tried to restore the backup in TeamPass in version 3.1.2, I couldn't solve it and I didn't get any feedback on the forum.

Today I installed version 3.1.3.1 and I still can't restore a backup.

Error:

**Next error occurred
Error: Malformed UTF-8 data**

Raw answer from server:
500
 Oops! ERROR NOT ALLOWED.
For security reason, you have been disconnected.
Click to [log in](https://newteampass.sefaz.al.gov.br/includes/core/logout.php?token=).

![erroRestaurarBackup](https://github.com/user-attachments/assets/c4dfa6ab-a24b-4a43-b8ad-004ac1446cbe)

@nilsteampassnet Can you help me?
Thanks

**Comment by vura** [comment]
@nilsteampassnet 
@corentin-soriano 

Can You help me? Thanks

**Comment by ArbitCode** [comment]
Hi is the issue fixed, I am also getting same error. 

Please share the steps if it fixed|

thanks

**Reply by nilsteampassnet** [reply]
@ArbitCode What release is used?
If not on 3.2.2.x branch then upgrade first.
No maintenance on other branches will be done.

**Reply by nilsteampassnet** [reply]
CLosed - Managed as a bug
